db/columns.py

```diff
@@ -356,30 +335,39 @@ def _check_type_option_equivalence(type_options_1, type_options_2):
     return False
 
 
-def retype_column_in_connection(table, connection, engine, column_index, new_type, type_options={}):
+def retype_column(table_oid, column_index, new_type, engine, type_options={}, connection_to_use=None, table_to_use=None):
+    if table_to_use is None:
+        table = tables.reflect_table_from_oid(table_oid, engine)
+    else:
+        table = table_to_use
     column = table.columns[column_index]
```

Comment:
```suggestion
   table = table_to_use
   if not table:
       table = tables.reflect_table_from_oid(table_oid, engine)
    column = table.columns[column_index]
```

I could see an argument to avoid the assignment, but I prefer setting a default and then changing it if needed. Just a nit, and feel free to ignore.

Reply:
Thanks, updated! I also updated it to be consistent in the other places where this was used.